Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -177,7 +177,7 @@ private void Initialize()
                 Utilities.Editor.InputMappingAxisUtility.CheckUnityInputManagerMappings(Definitions.Devices.ControllerMappingLibrary.UnityInputManagerAxes);
```

Comment:
Can you revert these changes to the MixedRealityToolkit class, as this does not resolve the issues in #2979 
PR #2981 pushed to resolve the underlying issue.

Reply:
If needed, I can include the "IsAssignableFrom" change, if it's felt this actually helps.

Reply:
Will do. And yes, I'd like us to use is assignable from.